stackage-server.cabal

```diff
@@ -90,9 +90,9 @@ library
 
     build-depends:
                     base                          >= 4.8         &&   < 4.9
-                  , aeson                         >= 0.8         &&   < 0.9
-                  , aeson-extra                   >= 0.2         &&   < 0.3
-                  , aws                           >= 0.12        &&   < 0.13
+                  , aeson                         >= 0.9         &&   < 0.10
+                  , aeson-extra                   >= 0.3         &&   < 0.4
+                  , aws                           >= 0.13        &&   < 0.14
```

Comment:
Not sure if it affects us:

> - DDB: !BREAKING! Add support for native boolean values via "Bool". Can read old values,
and there's a compatibility wrapper OldBool that behaves exactly the same way it
used to.